Fix grammar test (#272)
flyteorg/flytectl: docs/source/contribute.rst

###########################

First off, thank you for thinking about contributing! 

> Below you’ll find instructions that will hopefully guide you through how to contribute to, fix, and improve Flytectl.

Reviewer: ```suggestion
Here are the instructions that guide you through contributing, fixing, and improving Flytectl.
```
